packages/codecs-strings/src/base64.ts

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { combineCodec, Decoder, Encoder, mapDecoder, mapEncoder } from '@solana/codecs-core';
+
+import { getBaseXResliceDecoder, getBaseXResliceEncoder } from './baseX-reslice';
+import { InvalidBaseStringError } from './errors';
+
+const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/';
+
+/** Encodes strings in base64. */
+export const getBase64Encoder = (): Encoder<string> => {
+    if (__BROWSER__) {
```

Comment:
Is it the case that we can do…

```ts
if (__BROWSER__) {
    // What you have here.
} else if (__NODEJS__) {
    return Buffer.from(value, 'base64');
} else {
    // Your userspace implementation
}
```

Reply:
We've only got tests for the browser and node.js environments though so it won't test the baseXReslice implementation. 🤔